nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/status-history/status-history.component.html

```diff
@@ -14,7 +14,13 @@
 ~  See the License for the specific language governing permissions and
 ~  limitations under the License.
 -->
-<div resizable (resized)="resized()" [minHeight]="720" [minWidth]="887" class="flex flex-col status-history-container">
+<div
+    resizable
+    [resizeDisabled]="instances.length === 0 || fieldDescriptors.length === 0"
+    (resized)="resized()"
+    [minHeight]="720"
+    [minWidth]="887"
+    class="flex flex-col status-history-container">
```

Comment:
Not specific to your changes, but while you are in here... does it make sense to update the logic when deciding to show/hide the refresh/last loaded timestamp in the footer?
Currently it only bases its decision on the number of instances and ignores the number of fieldDescriptors so we end up with an odd dialog like so:
<img width="349" alt="Screenshot 2024-05-08 at 10 35 06 AM" src="https://github.com/apache/nifi/assets/713866/2d1aca73-f066-4743-ba88-d6b73b229763">

technically they can keep refreshing so it isn't terrible as it is. just not ideal. i suppose if they hit refresh and there happens to be data at that point, the dialog would just grown and look like the other.

what are your thoughts here, @scottyaslan?

Reply:
Sure, I can hide it in this case.